### DISCUSSION TITLE
broken evaluation queue

### DISCUSSION THREAD

**Discussion by JustLuckyRobotics** [cyberbotics/wrestling#60] [General]
current run trigger has been running for 4 hours with no print statements.

**Comment by omichel** [comment]
Yes, it seems something strange happened for [this run](https://github.com/cyberbotics/wrestling/actions/runs/4471549973/jobs/7856581016). Apparently it was automatically cancelled by GitHub after 6 hours. Not sure what was the problem...

**Comment by JustLuckyRobotics** [comment]
@omichel  i believe this issue has happened again, last time there was a double commit by the same person, looks like the same thing has happened here.

**Reply by omichel** [reply]
Strange. I am looking into this...
